README.md

```diff
@@ -173,6 +173,33 @@ To start a validator, you can also run the following command from the `snarkOS`
 ./run-validator.sh
 ```
 
+### 3.2.1 Enable Validator Telemetry Metrics (Optional)
+
+You can optionally enable validator telemetry to track participation in consensus. This is controlled via the telemetry feature flag and can be enabled in one of two ways:
+
+Once enabled, telemetry metrics are exposed through:
```

Comment:
Because of the previous sentence, I'd perhaps move this section to below the instructions on how to enable telemetry?